.freeipa-pr-ci.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
         git_refspec: '{git_refspec}'
         template: &ci-master-f27
           name: freeipa/ci-master-f27
-          version: 1.0.2
+          version: 1.0.3
```

Comment:
General comment: on the [vagrant cloud page](https://app.vagrantup.com/freeipa/boxes/ci-master-f27) the box does not have any description, hence it's difficult to know why a specific version has any added value compared to the preceding one.

Reply:
Thanks, the description makes sense.